Fun Commands/Mini-Games/Dyno-RPG/README.md

```diff
@@ -4,51 +4,44 @@ permalink: /FunCommands/MiniGames/DynoRPG/
 ---
 
 # Dyno RPG
-Otherwise called **Avelon RPG**, Dyno RPG is a RPG Game made using Dyno CC and inspired by the infection minigame and a few other custom commands. It is planned to be a community project to be built into something everybody can enjoy. As such everyone is welcome to add their own commands to it as they desire.
+**Dyno RPG** is a RPG Game made using Dyno Custom Commands and inspired by the infection minigame and a few other custom commands. It is planned to be a community project to be built into something everybody can enjoy. As such everyone is welcome to add their own commands to it as they desire.
 
 
 ## How to
```

Comment:
Yeah, it kinda is to an extent tbh. Dyno RPG is more or less designed to be user choice in the most part to make it as fun as possible. Some roles are strict but otherwise the game is more or less designed to have each roles limits chosen by the server staff.